elementary/monitor/api/alerts/alerts.py

```diff
@@ -0,0 +1,277 @@
+import json
+from collections import defaultdict
+from datetime import datetime
+from typing import Callable, Dict, List, Tuple
+
+from elementary.clients.api.api import APIClient
+from elementary.clients.dbt.dbt_runner import DbtRunner
+from elementary.config.config import Config
+from elementary.monitor.alerts.alert import Alert
+from elementary.monitor.alerts.alerts import Alerts, AlertsQueryResult
+from elementary.monitor.alerts.malformed import MalformedAlert
+from elementary.monitor.alerts.model import ModelAlert
+from elementary.monitor.alerts.source_freshness import SourceFreshnessAlert
+from elementary.monitor.alerts.test import TestAlert
+from elementary.monitor.api.alerts.normalized_alert import NormalizedAlert
+from elementary.utils.log import get_logger
+from elementary.utils.time import DATETIME_FORMAT, get_now_utc_str
+
+logger = get_logger(__name__)
+
+FALLBACK_TIME = datetime.fromtimestamp(0).strftime(DATETIME_FORMAT)
+
+
+class AlertsAPI(APIClient):
+    def __init__(
+        self, dbt_runner: DbtRunner, config: Config, elementary_database_and_schema: str
+    ):
+        super().__init__(dbt_runner)
+        self.config = config
+        self.elementary_database_and_schema = elementary_database_and_schema
+
+    def get_new_alerts(self, days_back: int, disable_samples: bool = False) -> Alerts:
```

Comment:
Could you maybe install a Linter for VSCode or use PyCharm?
It seems that here are typing issues in multiple places in the code.

<img width="667" alt="image" src="https://user-images.githubusercontent.com/30181361/209471256-7ba5afed-b6dc-4c20-a88f-23e392890bdf.png">